params/config.go

```diff
@@ -239,21 +239,44 @@ var (
 		},
 	}
 
+	// AleutChainConfig contains the chain parameters to run a node on the Aleut test network.
+	AleutChainConfig = &ChainConfig{
+		ChainID:             big.NewInt(133519467574834),
+		HomesteadBlock:      big.NewInt(0),
+		DAOForkBlock:        nil,
+		DAOForkSupport:      true,
+		EIP150Block:         big.NewInt(0),
+		EIP155Block:         big.NewInt(0),
+		EIP158Block:         big.NewInt(0),
+		ByzantiumBlock:      big.NewInt(0),
+		ConstantinopleBlock: big.NewInt(0),
+		PetersburgBlock:     big.NewInt(0),
+		IstanbulBlock:       big.NewInt(0),
+		MuirGlacierBlock:    nil,
+		BerlinBlock:         big.NewInt(0),
+		YoloV3Block:         big.NewInt(0),
+		AleutBlock:          big.NewInt(0),
```

Comment:
Has there been a testnet to test the transition to eip-1559?

Reply:
Yes, `aleut` is running currently. You can find the spec [here](https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/client-integration-testnets/aleut.md). Additionally, I've added an `--aleut` flag in this [commit](https://github.com/quilt/go-ethereum/pull/48/commits/55d2efc592cdafb0a54cf5c0f51c60f8e911c993) on a different branch.

Reply:
But then the chainconfig is wrong here?

Reply:
Yep, you're right. The `chainID` is hasn't been updated here.

Reply:
Also the aleut block should be 10 I think